tests/Fold.hs

```diff
@@ -85,7 +85,10 @@ suite = do
   -- "Gauge Pollster" resources
   describe "Folding points for POLLSTER resource: INSTANCE FLAVOR" $
     it "ok for example payload"
-      $ L.fold foldInstanceFlavor flavorTimedPDs `shouldBe` M.fromList flavorTimedPDsResult
+      $ L.fold (foldInstanceFlavor $ const True) flavorTimedPDs `shouldBe` M.fromList flavorTimedPDsResult
+
+  describe "Filtering instance statuses is sane" $
```

Comment:
This use of "describe" is odd - normally, describe [is used to name the thing which is described by the tests](https://hspec.github.io/), not assert that the thing is sane.